i18n/en_frontend.json

```diff
@@ -1,13 +1,14 @@
 {
   "frontend": {
-    "adjusting_focus": "Adjusting Focus..",
+    "adjusting_focus": "Adjusting focus..",
     "template_saved": "Template saved successfully",
-    "print_in_progress": "Printing in Progress...",
-    "printer_error": "Printer Error!",
+    "print_in_progress": "Printing in progress...",
+    "printer_error": "Printer error!",
```

Comment:
I think that the capitalization here is correct. We chose to go with "title capitalization", since the text is short and not a complete sentence. (https://www.grammarly.com/blog/capitalization-in-the-titles/)

Reply:
Okay, I'll reverse those changes.

I know this is the small details here now but:

* I'd add a third dot in "Adjusting Focus..."
* "Template saved successfully", "Please check your camera" and "No USB Storage found" do not end with a dot but "Error writing to USB storage." does.

I'm not sure how to make it consistent. But it's also not a big deal. 😁

Reply:
You are right, it would be nice to have it consistent.

Maybe like this:
- If it's a complete sentence (e.g. "Template saved successfully."), we end with a dot and capitalize normally
- If it's not a complete sentence (e.g. "Low Storage" or "Cancel Printing?") we use title capitalization and omit the dot.

No "USB Storage found" should then be "No USB storage found."

What do you think?